pkg/hoist/hoist_launchable_test.go

```diff
@@ -120,38 +97,11 @@ func TestNonStandardEntryPoint(t *testing.T) {
 	Assert(t).IsNil(err, "Error occurred when obtaining runit services for launchable")
 
 	expectedServicePaths := []string{
-		"/var/service/testPod__testLaunchable__start",
+		"/var/service/testPod__testLaunchable__bin__start",
 	}
 	Assert(t).AreEqual(1, len(executables), "Found an unexpected number of runit services")
 
-	for _, expected := range expectedServicePaths {
-		found := false
-		for _, executable := range executables {
-			if executable.Service.Path == expected {
-				found = true
-				break
-			}
-		}
-
-		if !found {
-			t.Errorf("Did not find %s among executable paths", expected)
-		}
-	}
-}
-
-// Tests that the Executables() function fails if two runit services with the same
-// service name will be created. We specify bin/start (a normal file) and bin/start2
-// (a directory containing one file called start) which will generate a naming
-// collision and thus (we hope) an error
-func TestErrorIfConflictingServiceNames(t *testing.T) {
```

Comment:
I suppose if you really wanted you could keep this best but instead now it should ensure it accepts conflicting service names?